core/cas-server-core-events/src/test/java/org/apereo/cas/support/events/HttpServletRequestSimulation.java

```diff
@@ -0,0 +1,84 @@
+package org.apereo.cas.support.events;
+
+
+import org.apereo.cas.mock.MockTicketGrantingTicket;
+import org.apereo.cas.support.events.ticket.CasTicketGrantingTicketCreatedEvent;
+import org.apereo.cas.util.HttpRequestUtils;
+import lombok.val;
+import org.apereo.inspektr.common.web.ClientInfo;
+import org.apereo.inspektr.common.web.ClientInfoHolder;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.mock.web.MockHttpServletRequest;
+import java.util.concurrent.Callable;
+
+
+/**
+ *  This is {@link HttpServletRequestSimulation}.
+ *
+ *   This Callable class simulates a thread that an application server would create for
+ *   incoming HttpServletRequests, and will publish a CasTicketGrantingTicketCreatedEvent
```

Comment:
Same comment on formatting javadoc elements.

Reply:
It should be updated now.